bigcodebench/evaluate.py

```diff
@@ -277,6 +277,12 @@ def stucking_checker():
     if not os.path.isfile(result_path):
         with open(result_path, "w") as f:
             json.dump(results, f, indent=2)
+    
```

Comment:
Maybe adding `if not os.path.isfile(pass_at_k_path):`?

Reply:
Actually, I think a better way is to check if at least the Pass@1 scores are the same and decide whether we need to rewrite the result_path and pass_at_k_path. Wdyt?

Reply:
yes, sounds good

Reply:
Thanks! I'll merge the PR after your update.

Reply:
Done